benchmarks/bench_ml.py

```diff
@@ -211,17 +218,34 @@ def evaluate(self, point):
         point_mapped = {}
         for param, val in point.items():
             point_mapped[param] = self.space[param][1](val)
+
+        # this is necessary to make things run in parallel
```

Comment:
This was a comment from some code experimentation I had earlier, I just forgot to update the comment :)
Relevant to this: the parallel computation did not work, I am not sure why, but it seems that it was due to same instance of estimator used in multiple processes. I switched to passing classes to threads instead of instances, and everything seems to work fine now.